tools/rosbash/rosbash

```diff
@@ -954,30 +954,98 @@ function _roscomplete_rosconsole {
     COMPREPLY=()
     arg="${COMP_WORDS[COMP_CWORD]}"
 
+    VERSION=$(rosversion rospy)
+    if [[ $? == 1 ]]; then
+        VERSION="0.0.0"
+    fi
+
+    VERSION_ECHO="1.13.7"
+    dpkg --compare-versions ${VERSION_ECHO} le ${VERSION}
```

Comment:
Can we rely on this to be available across all targeted platforms (e.g. Fedora, macOS)?

Reply:
Good call.

Honestly, I google how to reliably compare versions. I picked this option because it was really simple, but it probably isn't available in other OSs.

I'm open to suggestions. What do you think of the two options here: https://stackoverflow.com/questions/16989598/bash-comparing-version-numbers/24067243 ?

I think `sort -V` should work in most systems nowadays, although some people say it's only available in the GNU sort version. If you think it's better to perform the check w/o `sort -V`, I can implement the other option.

Reply:
Actually, it seems the FreeBSD supports `sort -V`. Do you know if macOS supports it? I don't have a Mac at hand atm.

Reply:
The man pages are online:

https://developer.apple.com/legacy/library/documentation/Darwin/Reference/ManPages/man1/sort.1.html

I don't see a `-V` option.

Reply:
Seems like a custom bash function is the most portable way to do it (without using Python):

https://stackoverflow.com/a/4025065/671658

Reply:
Actually, that online man page might be out-of-date, on my macbook I see the option:

```
% man sort | grep '\-V'
     -V, --version-sort
```

Reply:
@dirk-thomas @wjwwood I'm happy to create a new PR, but I'm not sure if you'd be ok with using `sort -V` or you prefer the approach from https://stackoverflow.com/a/4025065/671658 :thinking:

Reply:
> I'm happy to create a new PR

Absolutely, please make sure to target the default branch (which is `noetic-devel`) first and the patch can be backorder afterwards.

> but I'm not sure if you'd be ok with using sort -V or you prefer the approach from https://stackoverflow.com/a/4025065/671658

As long as it is proven to work on currently supported platforms anything is fine with me.